zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -988,12 +988,12 @@ impl<'a, F: Field> ConstraintBuilder<'a, F> {
     // Stack
 
     pub(crate) fn stack_pop(&mut self, value: Expression<F>) {
-        self.stack_lookup(false.expr(), self.stack_pointer_offset.expr(), value);
-        self.stack_pointer_offset += 1;
+        self.stack_lookup(false.expr(), self.stack_pointer_offset.clone(), value);
```

Comment:
This change is unnecessary, as the `self.stack_pointer_offset.expr()` will perform the same operation.

It's up to you whether you wish to revert this. But I personally prefer `.expr()` instead of `.clone()` over here because `Clone` does not tell me what type it'll be, but with `Expr`  I know it will be an expression. Up to you :)